.travis.yml

```diff
@@ -37,6 +37,10 @@ install:
       ansible-playbook -i inventory -vv --connection=local
       install_files/ansible-base/securedrop-development.yml
 script:
+  # For some reason, redis-server does not start automatically when installed
+  # on Travis. I believe Travis' service machinery may be interfering. See
+  # http://docs.travis-ci.com/user/database-setup/#Redis
+  - sudo service redis-server start
```

Comment:
This doesn't seem like this belongs here.

Reply:
Yes, it's a temporary commit fixing travis, the PR is https://github.com/freedomofpress/securedrop/pull/2273. Without it travis just fails to run.

Reply:
Fair enough. Approved.